src/View/globals.js

```diff
@@ -9,12 +9,25 @@
  * file that was distributed with this source code.
 */
 
+const GE = require('@adonisjs/generic-exceptions')
+
 module.exports = function (View, Route, Config) {
   /**
    * Return url for the route
    */
   View.global('route', function (...args) {
-    const url = Route.url(...args)
+    let url
+
+    try {
+      url = Route.url(...args)
+    } catch (error) {
+      if (error.message) {
```

Comment:
I believe error will always have a message, so the check is not required

Reply:
Yep. That's true. I've edited the file